src/renderer/shims/LocalShim.ts

```diff
@@ -81,12 +81,24 @@ export class LocalShim implements SimulatorInterface {
     public Aircraft = {}
 
     public GetStoredData = (key: string): any => {
-        if (key === 'A32NX_CONFIG_SELF_TEST_TIME') {
-            return 2;
+        // if (key === 'A32NX_CONFIG_SELF_TEST_TIME') {
+        //    return 2;
+        //}
+
+        try {
+            const item = window.localStorage.getItem(key);
+            return item ? JSON.parse(item) : null;
```

Comment:
Should return an empty string if the key doesn't exist, should not parse the string into an object.

Reply:
Reg. "should not parse the string into an object", Was just takeing the code from `LocalShim.GetSimVarValue(key: any)`
```
        GetSimVarValue(key: any) {
            try {
                const item = window.localStorage.getItem(key);
                return item ? JSON.parse(item) : 0;
            } catch (error) {
                console.log(error);
                return 0;
            }
        },
```
This returnes the parsed object. Why should it be different in `GetStoredData(key:string)`?

Reply:
GetSimVarValue can return different types, and isn't related to stored data. See reply above for why this should match the sim implementation.

Reply:
ok